### PR TITLE
Add description column to filters, and censored version of the ad filter

### PR DESCRIPTION
**Pull request by Davidy22**
Also ran black on frames.py, this also probably should get split up as the program gets bigger.

### CODE REVIEW

**Comment by Trisanu-007** [comment]
skunkbooth-fault
Could use some padding on the right side Ig. 
Also what's the thing with the ad filter?
Rest all looks good.

**Comment by Davidy22** [comment]
The filter from the video that I didn't commit, now censored

**Comment by Trisanu-007** [comment]
The padding is there, but the slight thickening of the border is also there.

**Comment by Davidy22** [comment]
Put a little row of . down the side for visual seperation, is there more than that you're seeing on the border?

**Comment by Trisanu-007** [comment]
The same right border in the picture above, but with a bit of padding this time.

**Comment by Davidy22** [comment]
Is padding not what we wanted?

**Comment by Trisanu-007** [comment]
padding seems fine, the right border looks different than the left one. The right is like the pic above and left is thin as usual.

**Comment by dhananjaylatkar** [comment]
> skunkbooth-fault
> Could use some padding on the right side Ig.
> Also what's the thing with the ad filter?
> Rest all looks good.

@Trisanu-007 why is Box aligned to left? Shouldn't it be centered?

**Comment by dhananjaylatkar** [comment]
Can we think of something witty instead of "**Everything is _**" for description?

**Comment by Trisanu-007** [comment]
Hmm yeah the alignments changed too, can you test on your machine too.

File: skunkbooth/utils/frames.py [hunk 1]
```diff
@@ -204,7 +204,7 @@ def __init__(self, screen: Any, filters: Any, data: Any = None) -> None:
             f[1] = temp
             logging.info(f"{f[0].name} button created")
             filters_layout.add_widget(temp)
-            filters_layout.add_widget(Label(f[0].description, align=">"), 1)
+            filters_layout.add_widget(Label(f"{f[0].description}   ]", align=">"), 1)
```
Comment (dhananjaylatkar) [inline skunkbooth/utils/frames.py:207]: `]` is looking put of place. I tried with unicode pipe symbol with one space and it looks much less intrusive.
```diff
-            filters_layout.add_widget(Label(f"{f[0].description}   ]", align=">"), 1)
+            filters_layout.add_widget(Label(f"{f[0].description} │", align=">"), 1)
```

**Comment by dhananjaylatkar** [comment]
Screenshot 2021-08-05 at 7 17 21 PM

Using pipe symbol with one space.

@Trisanu-007 i'm not seeing alignment problems

**Comment by Trisanu-007** [comment]
The right side is still messed up tho

**Comment by Davidy22** [comment]
What's the issue on the right side?

**Comment by Trisanu-007** [comment]
The right border doesn't look messed up?

**Comment by Davidy22** [comment]
It looks like a scrollbar. I guess the handle is a bit too small for a screen with nothing to scroll, but I think that's unconfigurable asciimatics stuff

**Comment by Trisanu-007** [comment]
In that case, this works. Ill merge it if that's okay?

**Comment by Davidy22** [comment]
Yeah sure

**Pull request merged**